src/components/header/renderer.tsx

```diff
@@ -153,10 +153,15 @@ class Header extends React.Component<Props, State> {
 
     const gistUrl = new URL(url, 'https://gist.github.com');
     const [username, gistId] = gistUrl.pathname.split('/').slice(1);
-
+    const gistCommits = await fetch(`https://api.github.com/gists/${gistId}/commits`).then(r => {
```

Comment:
One last thing. We can remove the `then` here and replace it with another await.

Reply:
It's done.